src/main/java/org/opentripplanner/graph_builder/GraphBuilderDataSources.java

```diff
@@ -181,6 +183,18 @@ public NetexFeedParameters getNetexConfig(DataSource dataSource) {
       .orElse(buildConfig.netexDefaults.copyOf().withSource(dataSource.uri()).build());
   }
 
+  /**
+   * Returns the optional data source for the stop consolidation configuration.
+   */
+  public Optional<DataSource> stopConsolidationDataSource() {
+    return Optional
+      .ofNullable(buildConfig.stopConsolidationFile)
+      .map(fileName -> {
+        var f = baseDirectory.toPath().resolve(fileName).toFile();
+        return new FileDataSource(f, FileType.CONFIG);
```

Comment:
This is not what we want, but we can fix it in another PR. The `CompositeDataSource#entry(String name)` is the thing we should use - of do the type thing, witch will use this.